test_scripts/Resumption/InteriorVehicleData/OnHashChange/001_OnHashChange_by_module_subscription_without_moduleId.lua

```diff
@@ -0,0 +1,43 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: SDL sends OnHashChange notification after successful subscription to interior vehicle data
+--  with default moduleId
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is not subscribed to modules
+--
+-- Sequence:
+-- 1. GetInteriorVD(subscribe = true, module_1) is requested
+-- SDL does:
+-- - a. send GetInteriorVD(subscribe = true, module_1, default moduleId) request to HMI
+-- - b. process successful responses from HMI
+-- - c. send OnHashChange notification to mobile app by receiving
+--    GetInteriorVD(subscribe = true, module_1, default moduleId)
```

Comment:
@GetmanetsIrina I think next phrase is redundant 
```
by receiving GetInteriorVD(subscribe = true, module_1, default moduleId
```
I would propose to remove it.
This comment relates to other scripts of the PR too.